stroyprombeton/models.py

```diff
@@ -182,6 +182,15 @@ def get_siblings(self, offset):
             .select_related('page')[:offset]
         )
 
+    # we'll remove this field
```

Comment:
Please create todo for this or note this in related issue

Reply:
@artemiy312 , left note at the comment already
https://github.com/fidals/stroyprombeton/issues/433#issuecomment-479831042